py/picca/io.py

```diff
@@ -1303,6 +1510,179 @@ def read_from_desi(nside,
 
     return data, num_data
 
+def read_from_minisv_desi(nside,
```

Comment:
can you check the names of the variables to match the rest of the functions? They seem to be the old names, prior to standardisation. The list of changes is here: https://docs.google.com/spreadsheets/d/1PougvkVWN_H5BubOCOs1ylkS0wnFSyDbOc3cvPEJXoc/edit?usp=sharing

Reply:
yes, some of them were, should be fixed now